components/history/CompletedTask.js

```diff
@@ -62,22 +64,27 @@ function CompleteTask(props) {
           )}
         </div>
 
-        <Typography variant="h6" component="h2" className={classes.group}>
+        <Typography component="h1" variant="h5" display="block" className={classes.group}>
           {title}
         </Typography>
-
-        <Typography variant="body2" component="p" className={classes.description}>
-          <Typography component="span" style={{ fontWeight: 'bold' }}>
-            Why?{' '}
+        <div data-intercom="task-why" className={classes.textBlock}>
```

Comment:
Do we need the `data-intercom` attr here? I don't think we currently have a product tour on this page.

Reply:
removed.